### PR TITLE
Selection

### PR DESCRIPTION
**Pull request by shayAlaluf**
separate map events manager and map selection service

### CODE REVIEW

File: src/services/map-events-mananger/consts/pickOptions.enum.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
- *  PICK_ONE    - in case a few entities are picked plonter is resolved . use Cesium.scene.drillPick()
  *  PICK_ALL    - all entities are picked. use Cesium.scene.drillPick()
+ *  MULTI_PICK  - every selected entity is picked. If chosen twice will be unselected. use Cesium.scene.pick()
  */
 export enum PickOptions {
 	NO_PICK,
 	PICK_FIRST,
-	PICK_ONE,
-	PICK_ALL
+	PICK_ALL,
```
Comment (eitanfr) [inline src/services/map-events-mananger/consts/pickOptions.enum.ts]: why pick_one was removed? 
pick_one and pick_first have different meanings .
also can remove multi_pick

Comment (shayAlaluf) [inline src/services/map-events-mananger/consts/pickOptions.enum.ts]: pick one had no implementation

Comment (eitanfr) [inline src/services/map-events-mananger/consts/pickOptions.enum.ts]: Check the original master version... To see the difference

Comment (shayAlaluf) [inline src/services/map-events-mananger/consts/pickOptions.enum.ts]: case PickOptions.PICK_ONE:
case PickOptions.PICK_ALL:
    picks = this.scene.drillPick(movement.endPosition);
    picks = picks.length === 0 ? null : picks;
    break;

it actually do pick_all, so i didn't understood why it needed.

**Pull request closed without merging**